src/Video/AbstractVideo.php

```diff
@@ -121,6 +126,11 @@ public function getAddPlayButton(): bool
         return $this->addPlayButton;
     }
 
+    public function isAutoplayEnabled(): bool
```

Comment:
Warum muss das in die abstrakte Klasse? Autoplay wird bei YouTube und Vimeo anders ausgewertet. In diesem Falls sollte es eigentlich in die konkrete Klasse (muss auch nicht nochmal extra ausgewertet werden, da alle get-Methoden) im Template-Kontext landen). Daher kann das auch aus der Generator-Klasse wieder raus.

Reply:
Ja hast recht, habs angepasst.